crypto/armor.go

```diff
@@ -7,6 +7,10 @@ import (
 	"io"
 
 	"github.com/cometbft/cometbft/crypto"
+	"github.com/google/tink/go/aead"
```

Comment:
if we use https://pkg.go.dev/golang.org/x/crypto@v0.7.0/chacha20poly1305, does it avoid importing a new library?